db_import/socrata_functions.R

```diff
@@ -49,6 +72,15 @@ create_sections_tables = function(
     
   }
   
+  # Coerce into characters
```

Comment:
I don't think this section is necessary.  I think that all of the columns are read from the json endpoint as character initially.  Can you confirm?

Reply:
All the columns are characters aside from `begindate` in both tables, which are POSIXct. I can update this section to cast only that field or leave it as is if we suspect that other columns could be interpreted as non-characters down the line.